### PR TITLE
Overhaul & Addition of CentCom Uniforms [MOSTLY MODULAR]

### PR DESCRIPTION
**Pull request by ChromeFoxxity**
## About The Pull Request

Adjusts almost all of CentCom sprites to a more vibrant, and softer green hue. As well as fixes berets to resemble the shading of the uniforms a bit more. Adds more stuff for gimmicks such as a Hazard Vest, and Labcoat, as well as minor additions to outfits, and adds some stuff to the Loadouts.

I have not fully tested it, but from what I know, it works as intended.

## Why It's Good For The Game

Makes the CentCom officials more diverse, and look even better while on-station. Adding new variants, and better colors for the outdated CentCom uniforms. 3 years of sitting collecting dust, and looking brand spanking new, as well as newer turtlenecks. CentCom looked out of place for a long while, so I changed it, and added some gimmick items for Admins to make it easier.

## Proof Of Testing

image

## Changelog

:cl:
add: CentCom Hazard Vest, Labcoat, Berets, Uniforms, Suits, Loadout Items.
fix: Fixed CentCom Sprites
image: Added Sprites for all new Uniforms, and newer designs for CentCom uniforms.
spellcheck: Fixed some grammar lackage.
code: Changed and Added code to some uniforms.
/:cl:

[Suit-less ERT Commander]
image

[Inspector]
image

[Commander]
image

[Medical Officer]
image

[Safety Inspector (OSHA)]
image

[Don't question the amount of commits, I still don't know how to undo that. Sorry]

### CODE REVIEW

**Comment by ChromeFoxxity** [comment]
Still don't know how to undo the last PR's commits, save me. :(

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -787,12 +787,14 @@
 /obj/item/clothing/head/beret/centcom_formal
 	name = "\improper CentCom Formal Beret"
 	desc = "Sometimes, a compromise between fashion and defense needs to be made. Thanks to Nanotrasen's most recent nano-fabric durability enhancements, this time, it's not the case."
-	icon_state = "beret_badge"
-	greyscale_config = /datum/greyscale_config/beret_badge
-	greyscale_config_worn = /datum/greyscale_config/beret_badge/worn
-	greyscale_colors = "#46b946#f2c42e"
+	icon_state = "centberet" // Bubberstation Edit
```
Comment (theselfish) [inline code/modules/clothing/head/jobs.dm:790]: You can easily make a lot of the clothing stuff modular just by redefining it in our modular files.

File: modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm [hunk 2]
```diff
@@ -124,7 +124,7 @@
 	new /obj/item/clothing/under/rank/nanotrasen_consultant(src)
 	new /obj/item/clothing/under/rank/nanotrasen_consultant/skirt(src)
 	new /obj/item/clothing/under/rank/centcom/officer(src)
-	new /obj/item/clothing/under/rank/centcom/officer_skirt(src)
+	new /obj/item/clothing/under/rank/centcom/officer/skirt(src)
```
Comment (theselfish) [inline modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm:127]: Upstream this.

File: modular_zubbers/code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -55,3 +55,81 @@
 	name = "security jacket"
 	desc = "A blue jacket in blue Peacekeeper colors. It has hi-vis stripes all over it."
 	icon_state = "secjacket_blue"
+
+/obj/item/clothing/suit/armor/vest/intern // Bubberstation Edit
```
Comment (theselfish) [inline modular_zubbers/code/modules/clothing/suits/armor.dm:59]: You don't need the edit comment in modular files.

Comment (ChromeFoxxity) [reply modular_zubbers/code/modules/clothing/suits/armor.dm:59]: > You don't need the edit comment in modular files.

Probably forgot to remove that-

File: code/modules/clothing/head/jobs.dm (hunk 1 above)
Comment (ChromeFoxxity) [reply code/modules/clothing/head/jobs.dm:790]: > You can easily make a lot of the clothing stuff modular just by redefining it in our modular files.

Fair enough, working on it.

File: modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm (hunk 2 above)
Comment (ChromeFoxxity) [reply modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm:127]: > Upstream this.

How do you mean?

File: modular_zubbers/code/modules/clothing/suits/armor.dm (hunk 3 above)
Comment (ChromeFoxxity) [reply modular_zubbers/code/modules/clothing/suits/armor.dm:59]: > You don't need the edit comment in modular files.

Fixed.

**Review by rye-rice — CHANGES_REQUESTED** [review]
Hey friend, it seems you took many sprites with zero credit. I'm assuming this is a mistake we wouldn't want to be plagiarizing, now would we?

Firstly, so many sprites are taken from 
https://github.com/tgstation/tgstation/pull/70226
that it is extremely disingenuous to not even credit it in the first place.

Secondly, it seems you ripped off some of the uniform sprites from
https://github.com/shiptest-ss13/Shiptest/pull/2748
as well, which also so happens to be from the same artist!

Seeing your previous track record of plagarizing so many times in the past, I doubt not crediting any of these are simple accidents. That being said, I have worked together with a maintainer; they will give you a chance to credit all the sprites you took with zero credit this time; including the ones I didn't catch as well. Just because people didn't catch some doesn't mean you didn't steal them as well. This pull request will be marked DNM until these issues are addressed.

**Comment by ChromeFoxxity** [comment]
> Hey friend, it seems you took many sprites with zero credit. I'm assuming this is a mistake we wouldn't want to be plagiarizing, now would we?
> 
> Firstly, so many sprites are taken from [tgstation/tgstation#70226](https://github.com/tgstation/tgstation/pull/70226) that it is extremely disingenuous to not even credit it in the first place.
> 
> Secondly, it seems you ripped off some of the uniform sprites from [shiptest-ss13/Shiptest#2748](https://github.com/shiptest-ss13/Shiptest/pull/2748) as well, which also so happens to be from the same artist!
> 
> Seeing your previous track record of plagarizing so many times in the past, I doubt not crediting any of these are simple accidents. That being said, I have worked together with a maintainer; they will give you a chance to credit all the sprites you took with zero credit this time; including the ones I didn't catch as well. Just because people didn't catch some doesn't mean you didn't steal them as well. This pull request will be marked DNM until these issues are addressed.

Yeah.. I'm just

**Comment by nikothedude** [comment]
lol. lmao, even

**Comment by ChromeFoxxity** [comment]
What are you lmaoing about? It was better to put it upstream instead.

**Comment by ChromeFoxxity** [comment]
> lol. lmao, even


Don't see why you have to mock me, I closed it to put it upstream instead for ease of use.

**Comment by nikothedude** [comment]
the way this pr was closed made it seem like you closed it in shame, sorry if it wasnt actually plagarism

**Pull request closed without merging**